src/main/java/uk/gov/hmcts/reform/managecase/repository/DefaultDefinitionStoreRepository.java

```diff
@@ -8,11 +8,14 @@
 import uk.gov.hmcts.reform.managecase.client.definitionstore.model.ChallengeQuestionsResult;
 
 import java.util.List;
+import java.util.Map;
+
+import static com.google.common.collect.Maps.newHashMap;
 
 @Repository
 public class DefaultDefinitionStoreRepository implements DefinitionStoreRepository {
 
-
+    public final Map<String, Integer> versions = newHashMap();
```

Comment:
Hashmap is not thread safe. Use concurrenthashmap please, if you need a shared map instance across requests.